src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -2113,12 +2113,6 @@ private static CSDiagnosticInfo GetFeatureAvailabilityDiagnosticInfo(SyntaxTree
                 return null;
             }
 
-            string requiredFeature = feature.RequiredFeature();
-            if (requiredFeature != null)
-            {
-                return new CSDiagnosticInfo(ErrorCode.ERR_FeatureIsExperimental, feature.Localize(), requiredFeature);
```

Comment:
>ErrorCode.ERR_FeatureIsExperimental [](start = 44, length = 35)

Is this error used elsewhere? Otherwise, consider commenting out the item and removing the corresponding message.